flowey/flowey_lib_hvlite/src/_jobs/build_and_publish_openhcl_igvm_from_recipe.rs

```diff
@@ -112,6 +123,16 @@ impl SimpleFlowNode for Node {
             }
         }));
 
+        if let Some(built_openvmm_hcl) = size_check_openvmm_hcl {
```

Comment:
the fact that you can accept a `artifact_dir: ReadVar<PathBuf>`, and then not populate it, is genuinely one of flowey's major warts / footguns. it would be really good to find some time to switch this infra over to instead use `artifact_dir: WriteVar<PathBuf>`, which would translate to the flowey compiler erroring-out if it detected a pipeline configuration that would result in a empty artifact.